src/analyze/replacements.ts

```diff
@@ -96,22 +151,34 @@ export async function runReplacements(
         continue;
       }
 
-      const mdnPath = getMdnUrl(replacement.mdnPath);
+      const mdnPath = replacement.mdnPath
```

Comment:
when can this be nullish? can this be what it was before?

same for line 171 (`docUrl`)

This also means the message on 161 goes back to what it was, and on 181